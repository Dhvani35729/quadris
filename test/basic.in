drop
right
drop
right
right
left
drop
clockwise
drop
right
clockwise
right
clockwise
drop
counterclockwise
drop
drop
drop