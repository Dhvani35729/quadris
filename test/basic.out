Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
J          
JJJ        
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
J          
JJJ        
           
           
           
           
           
           
           
           
           
           
           
J          
JJJ        
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 J         
 JJJ       
           
           
           
           
           
           
           
           
           
           
           
J          
JJJ        
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 OO        
 OO        
           
           
           
           
           
           
           
           
           
           
           
J          
JJJ        
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  OO       
  OO       
           
           
           
           
           
           
           
           
           
           
           
J          
JJJ        
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
   OO      
   OO      
           
           
           
           
           
           
           
           
           
           
           
J          
JJJ        
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
           
J  OO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
           
           
           
           
J  OO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 TTT       
  T        
           
           
           
           
           
           
           
           
           
           
           
J  OO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  TTT      
   T       
           
           
           
           
           
           
           
           
           
           
           
J  OO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 TTT       
  T        
           
           
           
           
           
           
           
           
           
           
           
J  OO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
           
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
           
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
S          
SS         
 S         
           
           
           
           
           
           
           
           
           
           
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
ZZ         
 ZZ        
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
ZZ         
 ZZ        
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 ZZ        
  ZZ       
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
  Z        
 ZZ        
 Z         
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
   Z       
  ZZ       
  Z        
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  ZZ       
   ZZ      
           
           
           
           
           
           
           
S          
SS         
 S         
 TTT       
J TOO      
JJJOO      
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
S          
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  L        
LLL        
           
           
           
           
           
           
           
S          
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
LL         
 L         
 L         
           
           
           
           
           
           
           
S          
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  L        
LLL        
           
           
           
           
           
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
ZZ         
 ZZ        
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 ZZ        
  ZZ       
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
  Z        
 ZZ        
 Z         
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
   Z       
  ZZ       
  Z        
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  ZZ       
   ZZ      
           
           
           
           
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
J          
JJJ        
           
           
  ZZ       
   ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  L        
LLL        
           
           
  ZZ       
   ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
LL         
 L         
 L         
           
           
  ZZ       
   ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
IIII       
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
IIII       
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 OO        
 OO        
IIII       
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  OO       
  OO       
IIII       
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  OO       
  OO       
IIII       
LL         
 LZZ       
 L ZZ      
IIII       
LL         
 L         
SL         
SSZZ       
 S ZZ      
 TTT       
J TOO      
JJJOO      
-----------
Next:      
ZZ 
 ZZ